metazoa/species/Homarus/Homarus_americanus/Homarus_americanus_gca018991925v1_about.md

```diff
@@ -0,0 +1,24 @@
+**About *Homarus americanus***
+-------------------------
+The American lobster (*Homarus americanus*) is a species of lobster
+found on the Atlantic coast of North America, chiefly from Labrador to
+New Jersey. It is also known as Atlantic lobster, Canadian lobster,
+true lobster, northern lobster, Canadian Reds, or Maine lobster. It
+can reach a body length of 64�cm (25�in), and a mass of over 20
```

Comment:
Encoded chars not displaying correctly in md "?".

Reply:
cool. thanks! fixed